k8s-deep-dive-charly-scratchpad/08c-logs-spam.md

```diff
@@ -20,10 +20,22 @@ Try to edit the logs query to specifically look at the calls made by the pod lis
 
 `index:main source:kubernetes.audit @usr.name:"system:serviceaccount:default:pod-lister"`{{copy}}
 
-Then click on "Analytics" in the logs view to display the log query as a metric.
+Then click on "Analytics" in the logs view to display the log query as a metric:
+![switch to Analytics button](./assets/logs_analytics.png)
+
 </details>
 
-## Fix the problem
+<details>
+<summary>Hint</summary>
+
+[Click here](https://app.datadoghq.com/logs/analytics?agg_m=&agg_q=%40usr.name&agg_t=count&analyticsOptions=%5B%22bars%22%5D&event&index=&live=true&query=%40http.url_details.path%3A%22%2Fapi%2Fv1%2Fpods%22&stream_sort=desc&viz=timeseries) to view all the requests on the `/api/v1/pods` endpoint grouped by the user. 
```

Comment:
Let's move this Hint up the above one where we give away who is the offender
`@usr.name:"system:serviceaccount:default:pod-lister"`